src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+
+{{ figure_markup(
+  image="incorrect-crossorigin-attribute-by-file-extension.png",
+  caption='Percent of incorrect crossorigin values segmented by file extension on mobile devices.',
+  description='Pie chart showing the percent of incorrect crossorigin values segmented by file extension on mobile devices. `woff2` is 29.9% likely. `woff` is 19.1% likely. `js` is 9.9% likely. `css` is 7.6% likely. `gif` is 7.2% likely. fonts.googleapis.com is 7.1% likely. `png` is 6.9% likely. `ttf` is 6.6% likely. `jpg` is 2.0% likely.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=277634745&format=interactive",
+  sheets_gid="699889350",
+  sql_file="consoleLog_incorrect_crossorigin_type.sql"
+  )
+}}
+
+More than half (63.6%) of the cases when the `crossorigin` attribute on the `rel="preload"` hint is either missing or incorrect, are linked to the preloading of font files, with a total of 14,818 instances across the dataset.
+
+#### Invalid `as` attribute
+
+The `as` attribute plays an important role when preloading your resources and getting this wrong may result in downloading the same resource twice. On most browsers, specifying an unrecognized `as` attribute will ignore the preload. The supported values are `audio`, `document`, `embed`, `fetch`, `font`, `image`, `object`, `script`, `style`, `track`, `worker` and `video`.
+
+There are 17,861 cases of unrecognized values, with the most frequent error being omitting it completely; while the most common invalid as values are `other` and `stylesheet` (the correct value is `style`).
+
+{{ figure_markup(
+  caption='Pages incorrectly used `as="stylesheet"` instead of `"style"`',
+  content="1,114",
+  classes="big-number",
+  sheets_gid="1681733418",
+  sql_file="consoleLog_unused_preload.sql"
+)
+}}
+
+When using an incorrect `as` attribute value—as opposed to unrecognized value, such as using `style` instead of `script`; the browser will duplicate the file download as the request won't match the resource stored in the preload cache.
+
+<p class="note">Note: While <code>video</code> is included in the spec, it isn't supported by any browser and would be treated as an invalid value and ignored.</p>
+
+#### Unused font files
+
+More than 5% of pages which preload font files, preload more font files than needed. When preloading font files, all browsers which support `preload` also support `.woff2`. This means that, assuming that the `.woff2` font files are available, it is not necessary to preload older formats, including `.woff`.
```

Comment:
```suggestion
More than 5% of pages which preload font files preload more font files than needed. When preloading font files, all browsers which support `preload` also support `.woff2`. This means that, assuming that the `.woff2` font files are available, it is not necessary to preload older formats, including `.woff`.
```

Reply:
Are you sure this comma should be removed?

Reply:
I think it should be removed. They are not separate clauses but one is a direct continuation of the other.